### PR TITLE
Add Ansible infrastructure docs (closes #6)

### LINKED ISSUE #6: Document how to use FOSSRIT/infrastructure
_Related to FOSSRIT/tasks#107._

---

Need more clear documentation about how to use the FOSSRIT/infrastructure repo. Mostly what playbooks are run and how to push new changes out to production.

### PR DESCRIPTION
**Pull request by jwflory**
This commit adds docs to explain how the FOSSRIT/infrastructure
repository is set up and managed. This is high-level information about
the git repo and how it works. It doesn't include specific information
about how to run the playbooks since I think that makes more sense to
include in the FOSSRIT/infrastructure README or contributing guidelines.

Closes #6.

Screenshot of Ansible docs page rendered locally

### CODE REVIEW

File: docs/infra/ansible.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+The FOSS@MAGIC Tech Team uses Ansible for system change management.
+Previously, no configuration management tool was used for our infrastructure.
+Some git commits go back to 2013 for a few scripts used back in the day, but the migration to Ansible started in September 2018.
+
+
+## Overview
+
+Ansible playbooks are run locally by a system administrator.
+A system administrator can run Ansible playbooks once a user and SSH key is set up for them on the host.
+ew SSH keys and users are defined in the `sshd` role (see [Roles](#roles)).
```
Comment (kennedy) [inline docs/infra/ansible.md]: `(see [Roles](#roles))` has a rendering issue. See screenshot

Comment (kennedy) [inline docs/infra/ansible.md]: `ew SSH keys` I think the sentence was incomplete?

Comment (jwflory) [reply docs/infra/ansible.md]: @kennedy The screenshot was outdated from what is committed. I noticed the same mistake earlier. :sweat_smile: I originally had written `(see [Roles](#roles)`

**Comment by jwflory** [comment]
Thanks @kennedy, corrected the typo in commit afca77f9d6429a6c45f130b16fddd9f974f7200e. :slightly_smiling_face:

File: docs/infra/ansible.md (hunk 1 above)
Comment (kennedy) [reply docs/infra/ansible.md]: Oh i see.

**Pull request merged**